### ISSUE TITLE
Link color fails color contrast test

### ISSUE THREAD

**Issue by sowasred2012** [canonical-web-and-design/ubuntu-copy-docs#13]
The color of the link (#0078CC) against the yellow background (#FAD542) fails the [color contrast check](https://webaim.org/resources/contrastchecker/) for WCAG level AA:

![image](https://user-images.githubusercontent.com/2376968/179985034-accec1e1-49cb-4f3a-87eb-f4fbde86eb50.png)
